### ISSUE TITLE
Final grade

### ISSUE THREAD

**Issue by ixamxkamii** [ixamxkamii/ecommerce-pattern-library#9]
I think overall my pattern library works pretty well. There are a few accessibility errors that haven't been corrected and am missing a few components on the website. But I have a helpful description with all (maybe almost all) components that I have made so that a team member could make a new page that follows the brand without even having a wireframe to start with.

I enjoyed the project and learn a lot. Even if the project isn't perfect it helped me find a better appreciation for web design and helped me understand way more then I did beforehand. I definitely see how helpful pattern libraries will be to me in the future.

---

*Style & design* — **2**

I think most of my pattern library is well designed and cohesive with the brand.

*Patterns* — **2**

I definitely know I don't have all the components but I designed more components then what I used on the website.

*Explanations* — **3**

I think my explanations are pretty detailed and teammates will know what every piece is used for.

*Accessibility* — **1**

I know that accessibility is a  pretty big weak point for my website. I am missing aria labels and alt tags but most of my contract is good and legible with the exception of a few banners.

*Semantics* — **2.5**

I feel most of my semantics were proper with the exception of using divs on everything instead of having some as sections

*Responsiveness* — **2**

I feel that overall my website and components are pretty responsive. Where I think I was lacking was in the type. Some of my type didn't scale down properly on smaller screens and I think that was my biggest downfall.

*Code Quality* — **2**

I think my semantics and indentation was pretty good but I feel like in some areas my code (specifically when using grids) was over complicated with too many divs inside of divs.

*Git & Commits* — **1**

I think I was definitely lacking in the commit category. I didn't do many commits and my messages could have been more descriptive.

---

15.5/24 — **65%**
